packages/zoe/test/unitTests/contracts/test-brokenContract.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from 'tape-promise/tape';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import bundleSource from '@agoric/bundle-source';
+
+import harden from '@agoric/harden';
+
+import { makeZoe } from '../../../src/zoe';
+// TODO: Remove setupBasicMints and rename setupBasicMints2
+import { setup } from '../setupBasicMints2';
+
+const automaticRefundRoot = `${__dirname}/brokenAutoRefund`;
+
+test.only('zoe - brokenAutomaticRefund', async t => {
+  t.plan(1);
+  // Setup zoe and mints
+  const { moolaR } = setup();
+  const zoe = makeZoe({ require });
+  // Pack the contract.
+  const { source, moduleFormat } = await bundleSource(automaticRefundRoot);
+  const installationHandle = zoe.install(source, moduleFormat);
+
+  const issuerKeywordRecord = harden({ Contribution: moolaR.issuer });
+
+  // 1: Alice tries to create an instance, but the contract is badly written.
+  zoe.makeInstance(installationHandle, issuerKeywordRecord).then(
```

Comment:
I think we should use `t.throws` here. Is there a reason not to?

Reply:
`makeInstance` is async. I couldn't find a way to `await` the result.  It doesn't throw until the test is done.

Reply:
t.rejects() works.